roles/deploy/tasks/deploy.yml

```diff
@@ -111,14 +111,14 @@
 
 # Precompile assets
 
-- name: precompile assets
+- name: precompile assets # noqa 301
```

Comment:
Since this is a newly built directory, we could test for `all.js` or similar before running this.

Reply:
We could use the `creates:` directive on this task to achieve that. :+1:

Reply:
I've changed my mind on this. I don't think we should couple this task to a specific file existing or not in a specific path, it means if the asset pipeline changes in the future deployments will break...